test/runtests.jl

```diff
@@ -24,6 +24,13 @@ cd(@__DIR__)
     @test isfile(ParallelTestRunner.get_history_file(ParallelTestRunner))
 end
 
+@testset "subdir use" begin
+    d = pwd()
```

Comment:
```suggestion
    d = @__DIR__
```
otherwise tests depend on working directory.